src/game/Server/SharedDefines.h

```diff
@@ -162,7 +162,7 @@ enum Powers
     POWER_FOCUS                         = 2,         ///< Used by hunters after Cataclysm (4.x)
     POWER_ENERGY                        = 3,         ///< Used by rouges to do their spells
     POWER_HAPPINESS                     = 4,         ///< Hunters pet's happiness affect their damage
-    MAX_POWERS                          = 5,
+	POWER_RUNE                          = 5,
```

Comment:
MAX_POWERS needs to be re-added and POWER_RUNE removed, as well as tab/space correction